### PR TITLE
task 9 delete

### PR DESCRIPTION
**Pull request by Henry-WL**
task 9 delete

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -289,11 +289,25 @@ describe("8. PATCH /api/articles/:article_id votes", () => {
   });
 });
 
-describe("9. DELETE /api/comments/:comment_id returns 204", () => {
+describe.only("9. DELETE /api/comments/:comment_id returns 204", () => {
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:292]: Please remove all .only from all your tests. Also this commit should only contain the tests for this task.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -289,11 +289,25 @@ describe("8. PATCH /api/articles/:article_id votes", () => {
...
   describe("/api/comments/:comment_id", () => {
     describe("DELETE /api/comments/:comment_id returns 204", () => {
       test("status code: 204", () => {
-        return request(app).delete("/api/comments/1/").expect(204);
+        // Article 9 has comment IDs 1, 17 after deleting should only have comment_id 1
+        return request(app).delete("/api/comments/17/").expect(204).then(() => {
+          request(app).get("/api/articles/9/comments").expect(200).then(({body}) => {
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:298]: Once the delete request is successful, why would you need to double-check it?

File: __tests__/app.test.js [hunk 3]
```diff
@@ -289,11 +289,25 @@ describe("8. PATCH /api/articles/:article_id votes", () => {
...
+                "comment_id": 1,
+                "body": "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+                "article_id": 9,
+                "author": "butter_bridge",
+                "votes": 16,
+                "created_at": "2020-04-06T12:17:00.000Z"
+              }
+            ])
+          })
+        });
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:310]: Please add two more tests, for invalid and none-existent ID.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hey Henry, please see my comments and refactor your code.
